CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [User/Customize filter] Fix the custom filter of the `template` field
```

Comment:
Add the issue link and the PR link like this: `- <message> <issue> (<PR>)`

Example:
https://github.com/vatesfr/xen-orchestra/blob/58dc3244be6d40cb515bf243f01f3f3885382104/CHANGELOG.md#L9